tests/testkit_extras/helpers.rs

```diff
@@ -206,43 +209,35 @@ pub fn listunspent_entry(raw: &RawBitcoinTx, addr: &btc::Address, confirmations:
     })
 }
 
-pub fn block_hash_on_height(sandbox: &Sandbox, height: Height) -> Hash {
-    let blockchain = sandbox.blockchain_ref();
-    let snapshot = blockchain.snapshot();
-    let schema = Schema::new(&snapshot);
-    schema.block_hashes_by_height().get(height.0).unwrap()
-}
-
 /// Anchor genesis block using funding tx
-pub fn anchor_first_block(sandbox: &AnchoringSandbox) {
-    let client = sandbox.client();
+pub fn anchor_first_block(testkit: &mut AnchoringTestKit) {
+    let requests = testkit.requests();
 
-    let anchoring_addr = sandbox.current_addr();
-
-
-    client.expect(vec![
-        confirmations_request(&sandbox.current_funding_tx(), 50),
+    let anchoring_addr = testkit.current_addr();
+    requests.expect(vec![
+        confirmations_request(&testkit.current_funding_tx(), 50),
         request! {
             method: "listunspent",
             params: [0, 9_999_999, [&anchoring_addr.to_base58check()]],
             response: [
-                listunspent_entry(&sandbox.current_funding_tx(), &anchoring_addr, 50)
+                listunspent_entry(&testkit.current_funding_tx(), &anchoring_addr, 50)
             ]
         },
+        get_transaction_request(&testkit.current_funding_tx()),
     ]);
 
-    let hash = sandbox.last_hash();
+    let hash = testkit.last_block_hash();
     let (_, signatures) =
-        sandbox.gen_anchoring_tx_with_signatures(Height::zero(), hash, &[], None, &anchoring_addr);
-    let anchored_tx = sandbox.latest_anchored_tx();
-    sandbox.add_height(&[]);
+        testkit.gen_anchoring_tx_with_signatures(Height::zero(), hash, &[], None, &anchoring_addr);
+    let anchored_tx = testkit.latest_anchored_tx();
+    testkit.create_block();
 
-    sandbox.broadcast(signatures[0].raw());
-    client.expect(vec![
-        confirmations_request(&sandbox.current_funding_tx(), 50),
+    testkit.mempool().contains_key(&signatures[0].hash());
```

Comment:
You've seemingly missed `assert!` macro here and some other places in the 3d7a86532c28b6b197fdd42cef2a84582c41f1d1 fix.